recipes/stumpy/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "stumpy" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce2f46e7645d2a61d9a4f11a957b3b9bd833986ced44a9563332edc77229685c
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

This is a pure-Python package. This can be `noarch` and the version constraint can be specified below.
```suggestion
  noarch: python
```

---

Done